### PR TITLE
Create BlockCobblestone.java

### PR DESCRIPTION
**Pull request by WalshyDev**

### CODE REVIEW

**Comment by sakofchit** [comment]
Hey,

Before I merge it, do you mind commenting bits of it? It'll help us out a bunch.

Thanks!

File: minecraft/src/main/java/net/cubition/minecraft/block/BlockCobblestone.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import net.cubition.api.item.ItemBase;
+import net.cubition.api.item.ToolItem;
+import net.cubition.api.network.NetworkIDResource;
+import net.cubition.minecraft.tool.ToolPickaxe;
+
+public class BlockDirt extends ItemBase implements BlockType, NetworkIDResource {
+
+	@Override
+	public String getNetworkID() {
+		return "3";
```
Comment (WalshyDev) [inline minecraft/src/main/java/net/cubition/minecraft/block/BlockCobblestone.java:13]: Not 100% sure what this even is, got it from the other class.

File: minecraft/src/main/java/net/cubition/minecraft/block/BlockCobblestone.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	}
+
+	@Override
+	public float getDigSpeed(ToolItem item) {
+		if (item.getToolType() instanceof ToolPickaxe)
+			return 4;
+		else
+			return 2;
+	}
```
Comment (WalshyDev) [inline minecraft/src/main/java/net/cubition/minecraft/block/BlockCobblestone.java:22]: Setting speed, 4 if pick. 2 if anything other.

File: minecraft/src/main/java/net/cubition/minecraft/block/BlockCobblestone.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+	@Override
+	public String getTranslatableName() {
+		return "tile.cobblestone.default.name";
+	}
```
Comment (WalshyDev) [inline minecraft/src/main/java/net/cubition/minecraft/block/BlockCobblestone.java:27]: Get the name of the block

**Comment by WalshyDev** [comment]
Added

**Comment by Xesau** [comment]
@Acharige Sure he does not have to document these things. There is documentation already in the parent classes.

**Comment by sakofchit** [comment]
Heh, I meant commenting the code, but thanks anyways :P

**Pull request merged**